src/unix/async.c

```diff
@@ -281,6 +281,11 @@ static int uv__async_start(uv_loop_t* loop) {
      * it's just a placeholder and magic number which is going to be closed
      * during the cleanup, as other FDs. */
     err = uv__open_cloexec("/dev/null", O_RDONLY);
+    if (err < 0)
+        /* In the rare case that "/dev/null" isn't mounted open "/"
+         * instead.
+         */
+        err = uv__open_cloexec("/", O_RDONLY);
```

Comment:
What I mean is, the diff can just be this, right?
```suggestion
    err = uv__open_cloexec("/", O_RDONLY);
```

Reply:
@bnoordhuis Yes, You are right